### ISSUE TITLE
Assignment 3

### ISSUE THREAD

**Issue by ixjlyons** [moorepants/resonance#76]
Some general ideas left over from last week:

- generate a configuration plot and/or animation
- use an interactive plot to find something
- trifilar pendulum

Concepts to include:

- compound pendulum
- center of percussion
- clock pendulum
- friction

Ideas:

- Make a configuration plot of the system used to find the axial moment of inertia of a bike wheel. Use a small circle for the fulcrum and a circle with an appropriately thick border to represent the wheel.
- ~~Use interactive plot to determine the length of a simple pendulum that will match the oscillation of a compound pendulum.~~ (already done in in-class notebook)
- Use `curve_fit` to find the center of percussion of a compound pendulum. Need to provide compound pendulum params and simple pendulum solution form.
- Re-create the video of the person manually finding the center of percussion using a configuration plot animation. Maybe just run the animation twice with a matched and non-matched simple pendulum length.

**Comment by moorepants** [comment]
# Creating an animation

Create and animation of the bicycle wheel compound pendulum. At minimum it should have a wheel shape and some pivot point. Make as complex as you like. Point them to the matplotlib animation examples for some ideas.

# Trifilar Pendulum

Add this model to `resonance/linear_systems` and make a new release. Have them do some research to find the relationship between period and inertia of the object them selves. All of the parameters in eq 1.78 except T should be system constants. They will need to translate our parameters into whatever equation they find. This will need a figure. I can sketch one on my pad if needed. Put the parameter J in the canonical_coefficients method instead of making it a known constant (hide it). Have them figure out the inertia of the unknown object using their period estimation function and a curve fit.

# Center of Percussion

Give them some geometry and mass of a tennis racket shape. A hoop and a pole. Estimate the moment of inertia of the object and find the distance from where you hand would hold the racket and the "sweet spot" (center of percussion).

# Stiffness

Point them to some online materials on the stiffness of springs in series and in parallel. Have them calculate the equivalent stiffness of a combination of springs. If you add a simple mass/spring/damper to resonance, you could ask them to simulate the vibration with this compound spring and report the natural frequency. Once again point them torwards some materials, but they have to read and investigate them selves how to do this.

**Comment by moorepants** [comment]
# Estimate mass or stiffness from vibration

If you set up a system with an unknown mass and stiffness and they know the natural frequency relationship they can determine the unknown from their simulation.

**Comment by moorepants** [comment]
# Measurement of acceleration or rate gyro

Can they setup a measurement that simulated putting a 2 axis accelometer or 1 axis rate gryo on the book (or other system). Maybe they can compare their measurement with something. They could do an iphone on a cup vibrating and use an app to collect the acc and rate data, then try to recreate the simulation. Maybe the chips are not at COM of phone.

**Comment by moorepants** [comment]
For the last one you could have them try to find the location of the accelerometer. If they record acceleration and then compare it to the acceleration from the simulation and a parameter is the distance from the centroid to the accelerometer, you can then locate it. Maybe there is a spec sheet that shows the actual position.
